docs/samples/browser-plugin-meetings/index.html

```diff
@@ -48,10 +48,14 @@ <h2 class="collapsible">Meeting Management</h2>
 
               <form id="credentials">
                 <p class="note">
-                  NOTE: Click <a href="https://developer.webex.com/docs/getting-started" target="_blank">here</a> for access token
+                  NOTE: Click <a href="https://developer.webex.com/docs/getting-started" target="_blank">here</a> for
+                  access token
                 </p>
                 <div>
-                  <input id="access-token" name="accessToken" placeholder="Access Token" value="" type="text" required>
+                  <input id="guest-name" name="guestName" placeholder="Guest Name" value="" type="text">
```

Comment:
is it ok if user leaves the guest name blank? should we maybe pre-populate the edit box with "guest"?

Reply:
thats a good idea to prepopulate it with some name